### PR TITLE
V1

### PR DESCRIPTION
**Pull request by antoinerichez**

### CODE REVIEW

File: src/BikeDataProject.API/Models/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace BikeDataProject.API.Models
+{
+    public static class Constants
+    {
+        public static readonly string MobileAppUserAgent = "mobileapp";
```
Comment (MeurillonGuillaume) [inline src/BikeDataProject.API/Models/Constants.cs:5]: Could use some creativity

**Review by MeurillonGuillaume — APPROVED** [review]

File: src/BikeDataProject.API/Models/Constants.cs (hunk 1 above)
Comment (antoinerichez) [reply src/BikeDataProject.API/Models/Constants.cs:5]: O

Comment (antoinerichez) [reply src/BikeDataProject.API/Models/Constants.cs:5]: K

Comment (MeurillonGuillaume) [reply src/BikeDataProject.API/Models/Constants.cs:5]: Give it some love plz

**Review by BertSchoovaerts — APPROVED** [review]

**Review by xivk — APPROVED** [review]

**Pull request merged**